### DISCUSSION TITLE
Creating a new note in a specific subfolder

### DISCUSSION THREAD

**Discussion by edoski** [chrisgrieser/shimmering-obsidian#138] [Q&A] [answered]
I've been using Shimmering Obsidian the last few days, and noticed that the create new note feature, when inside of a subfolder from search, isn't working for me.

I do `o <folder_name>` → Select the folder → Select "Create new Note in here."

But the new note is never created; anyone know how to fix this?

Thanks

**Comment by chrisgrieser** [comment, marked as answer]
Sorry, that's indeed a bug. I just fixed it with version 3.12.6.

That bug has actually been there for months, I really wonder how nobody mentioned it before 🙈

**Reply by edoski** [reply]
No worries man, glad to know you got it! Brilliant plugin, enjoy your coffee;)

**Reply by edoski** [reply]
Tried it out on my end still no luck:(

**Reply by chrisgrieser** [reply]
Weird, it works on my end now 🤔 

Could you open up a bug report? the bug report form asks for various kinds of information I need to figure out whats still causing an issue on your end

**Comment by edoski** [comment]
all done:)
